### ISSUE TITLE
[No 4K Support] Text, Icons and Graphs are broken on Debian 10.1 (Gnome3)

### ISSUE THREAD

**Issue by 825i** [oguzhaninan/Stacer#333]
OS: Debian 10.1
Resolution: 3840x2160
Scaling: 2x

I'm not sure if it's due to my screen resolution or scaling setting or because I am using Gnome3 or something else.

Issues:

* Text and icons, primarily on the 'services' window are broken. See below screenshots.
* Additionally, the icons on the left sidebar are too small and the line graphs on the Resources window are very small.

Screenshots:

https://ibb.co/pQ2VwMs
https://ibb.co/fd4CWfw

**Comment by alexmyczko** [comment]
It's because of your Scaling: 2x setting

**Comment by 825i** [comment]
Fair enough but how do I fix this to work properly with 4K resolutions? As it stands then it seems Stacer doesn't support 4K?
